src/Text/Pandoc/Options.hs

```diff
@@ -363,6 +363,7 @@ data WriterOptions = WriterOptions
   , writerBeamer           :: Bool       -- ^ Produce beamer LaTeX slide show
   , writerSlideLevel       :: Maybe Int  -- ^ Force header level of slides
   , writerChapters         :: Bool       -- ^ Use "chapter" for top-level sects
+  , writerParts            :: Bool       -- ^ Use "part" for top-level sects in LaTeX
```

Comment:
I don't really like having two independent booleans for chapters and parts.  It would be better to have a type that can be set to sections, chapters, or parts.  But perhaps this is the best approach for backwards compatibility.